### PR TITLE
Enable hodor filesharing to run multiple apps simultaneously

### PR DESCRIPTION
**Pull request by phillip**

### CODE REVIEW

File: hodor [hunk 1]
```diff
@@ -155,15 +156,17 @@ class Hodor
         end
 
         # Initialize unison container
-        system("#{@docker_prefix} docker run #{share_volumes_opts.join(" ")} -p 45678:45678 -d --name fileshare gansbrest/fs-base")
+        system("#{@docker_prefix} docker run #{share_volumes_opts.join(" ")} -p 45678 -d --name #{@fileshare_name} gansbrest/fs-base")
       end
+      
+      fileshare_port = `#{@docker_prefix} docker port #{@fileshare_name} 45678 2>&1`.strip!.split(':')[1]
```
Comment (gansbrest) [inline hodor]: seem like fileshare_port is duplicated in multiple places, is there a way to define it once?

**Comment by gansbrest** [comment]
Hi. Could you describe the problem you are solving with this request, so other people could read it later? Some use cases as well.

Maybe it makes sense to update Readme as well?

**Comment by phillip** [comment]
I removed the duplication. I don't think the Readme needs to be updated because I expected this to work inherently. My use case is building an application with a micro service architecture. 

    cd app1
    hodor run
    cd ../app2
    hodor run

This would share the same unison port for both apps and merge each directory into the other.

**Comment by gansbrest** [comment]
Ok, sounds good. Lets give it a shot! Thanks a lot

**Pull request merged**